auto-configurations/spring-ai-mcp-client/src/main/java/org/springframework/ai/autoconfigure/mcp/client/McpClientAutoConfiguration.java

```diff
@@ -144,8 +144,8 @@ public List<McpSyncClient> mcpSyncClients(McpSyncClientConfigurer mcpSyncClientC
 		if (!CollectionUtils.isEmpty(namedTransports)) {
 			for (NamedClientMcpTransport namedTransport : namedTransports) {
 
-				McpSchema.Implementation clientInfo = new McpSchema.Implementation(commonProperties.getName(),
-						commonProperties.getVersion());
+				McpSchema.Implementation clientInfo = new McpSchema.Implementation(
+						commonProperties.getName() + " - " + namedTransport.name(), commonProperties.getVersion());
```

Comment:
It would be good to move this logic to a method for reusability and consistency across.

Reply:
sure. It is addressed